source/slang/slang-ast-modifier.h

```diff
@@ -1585,6 +1585,24 @@ class DynamicUniformModifier : public Modifier
     SLANG_AST_CLASS(DynamicUniformModifier)
 };
 
+// Sharing IRInst data of AggType (Struct/Interface) with members.
```

Comment:
This shouldn't be needed. In general we should never be creating AST nodes that references IR constructs.